plugins/services/src/js/structs/Service.js

```diff
@@ -96,6 +96,12 @@ module.exports = class Service extends Item {
     };
   }
 
+  isDeleting() {
+    const env = this.get("env");
+
+    return env && env.SDK_UNINSTALL;
```

Comment:
Pleas move the framework specific implementation to the respective struct as this one should serve as the base for all kinds of services.